pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,36 +1,41 @@
 {
   "dependencies": {
+    "Microsoft.Net.Native.Compiler": "1.6.1",
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.2-servicing-25107-01",
-    "Microsoft.NETCore.Targets": "1.1.0",
-    "Microsoft.Win32.Primitives": "4.3.0",
-    "System.ComponentModel.EventBasedAsync": "4.3.0",
-    "System.Data.Common": "4.3.0",
-    "System.Diagnostics.Contracts": "4.3.0",
-    "System.Diagnostics.StackTrace": "4.3.0",
-    "System.IO.IsolatedStorage": "4.3.0",
-    "System.Net.Http.Rtc": "4.3.0",
-    "System.Net.NameResolution": "4.3.0",
-    "System.Net.Requests": "4.3.0",
-    "System.Net.Sockets": "4.3.0",
-    "System.Net.WebHeaderCollection": "4.3.0",
-    "System.Net.WebSockets": "4.3.0",
-    "System.Net.WebSockets.Client": "4.3.0",
-    "System.Numerics.Vectors.WindowsRuntime": "4.3.0",
-    "System.Reflection.Context": "4.3.0",
-    "System.Runtime.InteropServices.WindowsRuntime": "4.3.0",
-    "System.Runtime.Serialization.Json": "4.3.0",
-    "System.Runtime.Serialization.Primitives": "4.3.0",
-    "System.Runtime.Serialization.Xml": "4.3.0",
-    "System.Runtime.WindowsRuntime": "4.3.0",
-    "System.Runtime.WindowsRuntime.UI.Xaml": "4.3.0",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
```

Comment:
Unless you think now would be an appropriate time to update this and have a newer CoreCLR package dependency than UWP MP 5.2.3/5.3.1.